extensions/default/src/ViewerLayout/ResizablePanelsHook.tsx

```diff
@@ -270,53 +281,55 @@ const useResizablePanels = (
       return;
     }
     const { width: panelGroupWidth } = resizablePanelGroupElemRef.current.getBoundingClientRect();
-    setRightPanelExpandedWidth((size / 100) * panelGroupWidth - sidePanelExpandedInsideBorderSize);
+    setRightPanelExpandedWidth(
+      (size / 100) * panelGroupWidth - panelGroupDefination.shared.expandedInsideBorderSize
+    );
   }, []);
 
   return [
     {
       expandedWidth: leftPanelExpandedWidth,
-      collapsedWidth: sidePanelCollapsedWidth,
-      collapsedInsideBorderSize: sidePanelCollapsedInsideBorderSize,
-      collapsedOutsideBorderSize: sidePanelCollapsedOutsideBorderSize,
-      expandedInsideBorderSize: sidePanelExpandedInsideBorderSize,
+      collapsedWidth: panelGroupDefination.left.collapsedOffsetWidth,
```

Comment:
This was not an offset width before so I am surprised it is now. I think it should just be collapsedWidth.